mlrun/config.py

```diff
@@ -553,6 +553,13 @@
         # returned values will be packaged together as the tuple they are returned in. Default is False to enable
         # logging multiple returned items.
         "pack_tuples": False,
+        # In multi-workers run, only the logging worker will pack the outputs and log the results and artifacts.
+        # Otherwise, the workers will log the results and artifacts using the same keys, overriding them. It is common
+        # that only the main worker (usualy rank 0) will log, so this is the default value.
+        "logging_worker": 0,
```

Comment:
It is currently relates only to mpi you are right, but like you suggested, it might be relevant for other multi-worker runs. This is why I left the "mpi" name out of it. 

Secondly, I think it only relates to `mlrun.package` and it should be there. It is only applying the logging limitations for returning objects from a function, not for when users may call direct `context.log_artifact(...)` from some worker, and because it is for all multi-worker runs, it is in this section of the conf.